src/screens/explore/ExploreList/ExploreList.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react';
+import { FlatList, StyleSheet, RefreshControl, StyleProp, ViewStyle } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Card } from 'src/components';
+import { useAppDispatch } from 'src/store';
+import { getFilterSettings } from 'src/store/selectors';
+import { AsyncThunks } from 'src/store/thunks';
+import { ExploreListItem } from 'src/types';
+import { getAccommodationsListQuery } from 'src/utils';
+
+interface CardProps extends ExploreListItem {}
+
+interface ExploreListProps {
+  style: StyleProp<ViewStyle>;
+}
+
+const ExploreList = ({ style }: ExploreListProps) => {
+  const dispatch = useAppDispatch();
+  const filter = useSelector(getFilterSettings);
+  const [data, setData] = useState<CardProps[]>([]);
+  const [pageNumber, setPageNumber] = useState<number>(1);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [isRefreshing, setIsRefreshing] = useState<boolean>(false);
+
+  const fetchData = useCallback(async () => {
+    try {
+      const response = await dispatch(
+        AsyncThunks.getListOfAccommodations(
+          getAccommodationsListQuery({ ...filter, page: pageNumber })
+        )
+      );
+      if (response.payload?.success) {
+        setData((prevData) => [...prevData, ...(response.payload?.data as CardProps[])]);
```

Comment:
consider using a selector to get the accommodations list, you don't need `useState` here

Reply:
thank you for this comment. Remade it that way, not it's much more neat